yoti-sdk-api/src/main/java/com/yoti/api/client/spi/remote/call/identity/DigitalIdentityService.java

```diff
@@ -79,8 +79,26 @@ public ShareSession createShareSession(String sdkId, KeyPair keyPair, ShareSessi
         }
     }
 
-    public Object fetchShareSession(String sessionId) {
-        return null;
+    public ShareSession fetchShareSession(String sdkId, KeyPair keyPair, String sessionId)
+            throws DigitalIdentityException {
+        notNullOrEmpty(sdkId, "SDK ID");
+        notNull(keyPair, "Application Key Pair");
+        notNull(sessionId, "Session ID");
+
+        String path = pathFactory.createIdentitySessionRetrievalPath(sessionId);
+
+        LOG.debug("Requesting share session with ID '{}' at '{}'", sessionId, path);
```

Comment:
(nit) the `with ID` could be dropped (same below in the exception message)

Reply:
addressing it in the next PR ;)